src/styles/styles.scss

```diff
@@ -26,11 +26,6 @@
 		animation-name: pulse;
 		animation-duration: 1600ms;
 		animation-iteration-count: 4;
-
-		&:hover,
-		&:focus {
-			opacity: .6 !important;
```

Comment:
This is only when you have notifications and it is intentionally designed like this, as you should click it and act on your pending notifications.

Reply:
or wait, this is the over way around.... it was made transparent when you had notifications and then hovered it?
Fine by me

Reply:
Can you also drop the other `opacity: .6;` in this file which is overwritten by server css?
![Bildschirmfoto vom 2022-06-02 14-06-08](https://user-images.githubusercontent.com/213943/171625683-77b15f7d-8d24-42e9-9d2f-9abe2b647686.png)

Reply:
line 10 in this file